modules/pxp-module-puppet

```diff
@@ -194,8 +209,8 @@ module Pxp
       return run_result
     end
 
-    # Wait for the lockfile to be removed. If it hasn't after 30 seconds, give up.
-    def wait_for_lockfile(lockfile = '', check_interval = 0.1, give_up_after = 30)
+    # Wait for the lockfile to be removed. If it hasn't after 10 minutes, give up.
```

Comment:
Changes to a 10 minute limit to wait before giving up. Previously it would go until it had a successful run.